pkgs/epnix/support/opcua/default.nix

```
@@ -13,13 +13,13 @@
 }:
 mkEpicsPackage {
   pname = "opcua";
-  version = "0.10.0-dev";
+  version = "0.10.0";
   varname = "OPCUA";
 
   src = fetchFromGitHub {
     owner = "epics-modules";
     repo = "opcua";
-    rev = "3d10053";
+    rev = "v0.10.0";
     hash = "sha256-EQra8PesO7Rlhj+pBlAfiqh5yjJwRkuh7gbGziY58iI=";
```

The version changed, so the hash should have changed. Try setting this:

```suggestion
    hash = "";
```

And then try to build it. Nix will tell you what hash to put.